src/components/Footer/MailchimpForm.tsx

```diff
@@ -36,48 +30,20 @@ const CustomForm: FC<FormHooks<EmailFormFields>> = ({
           Receive the latest news about Threshold
         </BodyLg>
       </SectionTextContainer>
-      <Box
-        as="form"
-        // @ts-ignore
-        onSubmit={handleSubmit}
-        w="full"
-        maxW={{ base: "100%", md: "320px", lg: "512px" }}
-      >
-        <FormControl>
-          <ResponsiveStack columnReverse>
-            <Input
-              size="lg"
-              onChange={(e) => setEmail(e.target.value)}
-              type="email"
-              value={email}
-              bg="white"
-              placeholder="Enter an email"
-            />
-            <Button
-              size="lg"
-              px={6}
-              type="submit"
-              isLoading={status === "sending"}
-            >
-              Submit
-            </Button>
-          </ResponsiveStack>
-
-          {status === "error" && (
-            <FormHelperText
-              color="red.500"
-              // slice first 4 chars to scrub the mailchimp error
-              // example mailchimp error: "0 - Invalid Email"
-              dangerouslySetInnerHTML={{ __html: (message as string).slice(4) }}
-            />
-          )}
-
-          {status === "success" && (
-            <FormHelperText color="green.500">
-              Please check your email to verify and join the newsletter.
-            </FormHelperText>
-          )}
-        </FormControl>
+      <Box w="full" maxW={{ base: "100%", md: "320px", lg: "512px" }}>
+        <iframe
+          src="https://embeds.beehiiv.com/38f40972-ced2-4f8e-b92b-2030750a59d0?slim=true"
+          data-test-id="beehiiv-embed"
+          height="52"
+          width="100%"
+          frameBorder="0"
+          scrolling="no"
+          style={{
+            margin: 0,
+            borderRadius: "0",
+            backgroundColor: "transparent",
+          }}
+        ></iframe>
```

Comment:
Maybe let's extract this to a separate component eg. `SubscribeToNewsletter`. If we changed the newsletter provider again we would only have to change the implementation inside the `SubscribeToNewsletter` component.

Reply:
I'm not sure if it's neccessary to be honest after the changes made in 728c282e33e7d69c0017dac9e73efee6c7486701. I renamed the `MailchimpForm` component to `NewsletterSubscribeForm` for all the inputs and labels related to the subscription. Creating a new `SubscribeToNewsletter` component that will only have the `iframe` with input might produce unnecessary confusion since we also have `NewsletterSubscribe` form for this whole section. Additionally we don't use this form/input anywhere  else so we don't need to make it that separate. 

Either way take a look at changes in 728c282e33e7d69c0017dac9e73efee6c7486701 and let me know what you think and if you still think it's neccessary. I can still change it if you think it's worth it in the long term.

Reply:
I thought we got this code inside the page component, sorry. After changes in 728c282 make perfect sense! LGTM!